marketplacetests/marketplace/pages/base.py

```diff
@@ -137,6 +129,23 @@ def show_menu(self):
             lambda m: 'mkt-nav--visible' in body.get_attribute('class'))
         return NavMenu(self.marionette)
 
+    def tap_install_button_of_first_paid_app(self):
```

Comment:
I agree that it is less than intuitive, but going with your suggestion re-adds much of the duplication I was trying to remove. I don't like the fact that each test needs to:
- locate a paid app
- verify that the app isn't installed
- tap on the install button for the app

and that we need to include each of those in the test. I was trying to encapsulate all of that into a single helper method, but the problem is that we need the app name in the test afterwards, and we also end up at the SearchResults page, so I felt it made sense that our helper return that.

I will try a different version where the helper is a test method, rather than a page object method, which might be a bit better. If that is still not good then we can go with what you suggest above.


Seems